### PR TITLE
Adds PostgreSQL to the TravisCI build matrix

### PR DESCRIPTION
**Pull request by Willianvdv**
While working #802 I figured it would be handy to have a CI build using PostgreSQL. This database can be a bit pickier what SQL can be run and this CI build will warn us when we add something that can't be run on psql.

### CODE REVIEW

File: spec/support/star_wars/schema.rb [hunk 1]
```diff
@@ -304,7 +304,10 @@ def edge_nodes
 
     connection :newestBasesGroupedByFaction, BaseType.connection_type do
       resolve ->(obj, args, ctx) {
-        Base.order('sum(faction_id) desc').group(:faction_id)
```
Comment (Willianvdv) [inline spec/support/star_wars/schema.rb]: This query can't be run on psql, @esjee [fixed this](https://github.com/Willianvdv/graphql-ruby/pull/1#issue-238826519)

**Comment by Willianvdv** [comment]
@esjee it's kinda interesting to see this fix results in a different order on Rails 3.2 builds https://travis-ci.org/rmosolgo/graphql-ruby/builds/247524385?utm_source=github_status&utm_medium=notification

File: .rubocop.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ AllCops:
     - 'lib/graphql/language/parser.rb'
     - 'gemfiles/**/*'
     - 'tmp/**/*'
+    - 'vendor/**/*'
```
Comment (Willianvdv) [inline .rubocop.yml]: For some reason if I don't add this my postgresql build will fail on thousands of lint issues in the vendor folder.

File: graphql.gemspec [hunk 3]
```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "racc", "~> 1.4"
   s.add_development_dependency "rails"
   s.add_development_dependency "rake", "~> 11"
-  s.add_development_dependency "rubocop", "~> 0.45"
+  s.add_development_dependency "rubocop", "~> 0.49.1"
```
Comment (Willianvdv) [inline graphql.gemspec]: I'll remove this from this PR

File: .rubocop.yml (hunk 2 above)
Comment (rmosolgo) [inline .rubocop.yml]: that's fine, we can support setups where the bundle is installed locally

File: graphql.gemspec (hunk 3 above)
Comment (rmosolgo) [inline graphql.gemspec]: :+1:

**Comment by Willianvdv** [comment]
@rmosolgo this PR is good to go! I'll post a follow-up PR to fix #802 (or prove it's not an issue :D)

File: Gemfile [hunk 4]
```diff
@@ -4,7 +4,7 @@ gemspec
 
 gem 'ruby-prof', :platform => :ruby
 gem 'sqlite3', :platform => :ruby
+gem 'pg', :platform => :ruby
 gem 'pry'
 gem 'pry-stack_explorer', :platform => :ruby
-
```
Comment (esjee) [inline Gemfile]: Unintended change?

Comment (Willianvdv) [inline Gemfile]: Thanks, fixed it

**Comment by rmosolgo** [comment]
:tada: It will be great to avoid these postgres incompatibility issues in the future, thanks for taking care of this!

**Pull request merged**